aws/resource_aws_cognito_identity_pool.go

```diff
@@ -218,9 +225,19 @@ func resourceAwsCognitoIdentityPoolUpdate(d *schema.ResourceData, meta interface
 		params.SamlProviderARNs = expandStringList(v.([]interface{}))
 	}
 
+	if v, ok := d.GetOk("tags"); ok {
+		params.IdentityPoolTags = tagsFromMapGeneric(v.(map[string]interface{}))
```

Comment:
Is this still necessary here? Seems like the `setTagsCognito` function should be handling tags already.

Reply:
fixed in 4ed6457ff